src/ray/raylet/scheduling/cluster_resource_manager.cc

```diff
@@ -292,4 +292,22 @@ BundleLocationIndex &ClusterResourceManager::GetBundleLocationIndex() {
   return bundle_location_index_;
 }
 
+void ClusterResourceManager::UpdateNodeLabels(
+    const scheduling::NodeID &node_id,
+    const absl::flat_hash_map<std::string, std::string> &labels) {
+  if (labels.empty()) {
+    return;
```

Comment:
I don't think we should early return here. If labels are empty then we should just set node labels to empty.

Reply:
Same as above.
This is the semantics of 'update', so it won't be overwritten. If such a scenario arises in the future, I will create a new interface called 'ResetNodeLabels()'